packages/dx-react-grid-demos/src/demo-sources/core-action/bootstrap3/simple-action.jsx

```diff
@@ -1,33 +1,25 @@
-import * as React from 'react';
+import React, { useState } from 'react';
 import {
   PluginHost, Plugin, Getter, Action, Template, TemplatePlaceholder, TemplateConnector,
 } from '@devexpress/dx-react-core';
 
-export default class Demo extends React.PureComponent {
-  constructor(props) {
-    super(props);
+const Demo = () => {
```

Comment:
Can we expert without creating a new variable?

```jsx
export default () => { ... };
```

Reply:
Yes, we can, but it could make code less readable.

Reply:
In fact, some demos had released by this way. For example: `grid-immutability` demo.

Reply:
Did it.